vagrant_support/cron.yml

```diff
@@ -0,0 +1,45 @@
+- name: set cron mailto
+  cron:
+    env: yes
+    name: MAILTO
+    value: vagrant@localhost
```

Comment:
Does vagrant actually make this work, such that it somehow delivers the mail to you?